baselines/hfedxgboost/hfedxgboost/conf/base.yaml

```diff
@@ -0,0 +1,49 @@
+---
+defaults: 
+  - dataset: cpusmall
+  - clients: cpusmall_2_clients
+
+centralized: False
+device: "cpu"
+
+n_estimators_client: ${clients.n_estimators_client}
+task_type: ${dataset.task.task_type}
+client_num: ${clients.client_num}
+
+XGBoost:
+  _target_:  ${dataset.task.xgb._target_}
+  objective: ${dataset.task.xgb.objective}
+  learning_rate: .1
+  max_depth: ${clients.xgb.max_depth}
+  n_estimators: ${clients.n_estimators_client}
+  subsample: 0.8
+  colsample_bylevel: 1
+  colsample_bynode: 1
+  colsample_bytree: 1
+  alpha: 5
+  gamma: 5
+  num_parallel_tree: 1
+  min_child_weight: 1
+
```

Comment:
```suggestion

client_resources:
    num_cpus: 1
    num_gpus: 0.0
```

Reply:
For consistency with many other baselines, let's add this to the config and pass it to `start_simulation`. I'll add related comments below.